src/atomate2/vasp/sets/base.py

```diff
@@ -948,12 +945,13 @@ def _set_u_params(incar, incar_settings, structure):
     # investigation it was determined that this would lead to a significant difference
     # between SCF -> NonSCF even without Hubbard U enabled. Thanks to Andrew Rosen for
     # investigating and reporting.
-    if "LMAXMIX" not in incar_settings.keys():
```

Comment:
@esoteric-ephemera: Can you undo this change re: the LMAXMIX logic? See https://github.com/materialsproject/pymatgen/pull/3041#issuecomment-1650336881 for details.